File: problems/168-excel-sheet-column-title/excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder res=new StringBuilder();
        
        while (columnNumber>0){
            columnNumber--;
            int rem=columnNumber%26;
            columnNumber/=26;
            res.insert(0,(char)(rem + 'A'));
        }
        
        return res.toString();
    }
}

File: problems/168-excel-sheet-column-title/excel-sheet-column-title.py
class Solution(object):
    def convertToTitle(self, c):
        s=''
        while c>0:
            if c%26>0: 
                s+=chr(64+(c%26))
                c//=26
            else: 
                s+='Z'
                c=(c//26)-1
        return s[::-1]